distro/configuration/patientidentifiertypes/patientidentifiertypes.csv

```diff
@@ -1,10 +1,3 @@
 Uuid,Void/Retire,Name,display:en,display:km,Description,Required,Format,Format description,Validator,Location behavior,Uniqueness behavior
 05a29f94-c0ed-11e2-94be-8c13b969e334,,OpenMRS ID,OpenMRS ID,លេខសម្គាល់ បើកប្រព័ន្ធកត់ត្រាអ្នកជំងឺ (MRS),"OpenMRS patient identifier, with check-digit",TRUE,,,org.openmrs.module.idgen.validator.LuhnMod30IdentifierValidator,NOT_USED,UNIQUE
-22348099-3873-459e-a32e-d93b17eda533,,Legacy ID,Legacy ID,លេខសម្គាល់ចាស់,"Identifier type to record optional previous identifers ",FALSE,,,,NOT_USED,
-b4143563-16cd-4439-b288-f83d61670fc8,,ID Card,ID Card,លេខអត្តសញ្ញាណប័ណ្ណ,ID Card,FALSE,,,,NOT_USED,UNIQUE
-a71403f3-8584-4289-ab41-2b4e5570bd45,,SSN,SSN,លេខសំគាល់សន្តិសុខសង្គម,Social Security Number,FALSE,[A-Z]{1}-[0-9]{7},"Identifier should be 1 letter, followed by a dash and 7 numerical characters. Eg, A-0010902",,NOT_USED,UNIQUE
-aeccdb8e-0766-4150-98cd-9205bdd575d5,,National ID,National ID,អត្ថ​សញ្ញាណ​ប​ណ្ណ​ជាតិ,National ID Number,FALSE,[0-9]{9},"National ID Identifier should be 9 numbers letter. Eg, 923456678",,NOT_USED,UNIQUE
-e23cdf5f-71d3-4fcf-9937-de689c5b1e9a,,COVID vaccination ID,COVID vaccination ID,លេខសម្គាល់ការចាក់វ៉ាក់សាំង COVID,COVID vaccination ID Number,FALSE,[0-9]{3}-[0-9]{3}-[0-9]{4},"COVID Identifier should be 10 numbers, separated by dashes. Eg, 321-123-1234",,NOT_USED,UNIQUE
-177c01be-789a-4c47-9515-3e6b092d4322,,IDPoor,IDPoor,លេខសម្គាល់ប័ណ្ណក្រីក្រ (ប័ណ្ឌមូលនិធិសមធម៌),IDPoor (Equity Card number),FALSE,[0-9]{8}-[0-9]{4}|[0-9]{8}-5[0-9]{3}|[0-9]{8}-9[0-9]{3}|[0-9]{3}-[0-9]{3}-[0-9]{3}-[0-9]{1}|[0-9]{7}-[0-9]{7}-[0-9]{1},,,NOT_USED,UNIQUE
-9cc705da-cd85-44b6-b4cf-05d6a55ae077,,NSSF,NSSF,បេឡាជាតិសន្តិសុខសង្គម,NSSF,FALSE,[0-9]{7}-[0-9]{7}-[0-9]{1},"NSSF Identifier should be 15 numbers, separated by dashes. Eg, 2343453-1234567-9",,NOT_USED,UNIQUE
-a4949ae8-d858-444b-ac9f-33c1d8825442,,PMRS ID,PMRS ID,លេខសម្គាល់ប្រព័ន្ធកត់ត្រា និងគ្រប់គ្រងអ្នកជំងឺអេឡិចត្រូនិក,PMRS patient identifier,FALSE,[0-9]{3}-[0-9]{3}-[0-9]{3}-[0-9]{1},"PMRS Identifier should be 10 numbers, separated by dashes. Eg, 321-123-123-4",,NOT_USED,UNIQUE
\ No newline at end of file
+22348099-3873-459e-a32e-d93b17eda533,,Legacy ID,Legacy ID,លេខសម្គាល់ចាស់,"Identifier type to record optional previous identifers ",FALSE,,,,NOT_USED,
```

Comment:
I don't think we even want this "Legacy ID" at all.
We should just remove it too.